.goreleaser.yml

```diff
@@ -56,3 +56,18 @@ dockers:
     extra_files:
       - etc/chart_schema.yaml
       - etc/lintconf.yaml
+brews:
+  - github:
+      owner: helm
+      name: homebrew-tap
+    commit_author:
+      name: helm-bot
+      email: helm-bot@users.noreply.github.com
+    folder: Formula
+    homepage: https://github.com/helm/chart-testing/
+    description: Testing and linting Helm charts
+    install: |
+      bin.install "ct"
+      etc.install "etc" => "ct"
```

Comment:
Had to look up `etc.install` https://www.rubydoc.info/github/Homebrew/brew/Formula:etc interesting!

Reply:
Me too. Took me a while to figure out how to do it. My first attempt was to copy it to `$HOME/.ct`, but it doesn't seem to be possible to add stuff to a user's home directory. Homebrew changes the complete environment during the installation.